anima-jws-voucher.mkd

```diff
@@ -189,20 +169,21 @@ The following figure provides an example of JSON Voucher Data:
 
 ## JWS Protected Header
 
-The JWS Protected Header defined in {{RFC7515}} uses the standard header parameters "alg", "typ", and "x5c".
-The "alg" parameter MUST contain the algorithm type used to create the signature, e.g., "ES256" as defined in {{Section 4.1.1 of RFC7515}}.
-If present, the "typ" parameter SHOULD contain the value "voucher-jws+json" as defined in {{Section 4.1.9 of RFC7515}}.
-If X.509 (PKIX) certificates {{RFC5280}} are used, the "x5c" parameter SHOULD contain the base64-encoded (not base64url-encoded) X.509 v3 (DER) certificate and chain as defined in {{Section 4.1.6 of RFC7515}}.
+The JWS Protected Header defined in {{RFC7515}} uses the standard header parameters `alg`, `typ`, and `x5c`:
+
+* The `alg` parameter MUST contain the algorithm type (e.g., `ES256`) used to create the signature as defined in {{Section 4.1.1 of RFC7515}}.
+* The `typ` parameter is optional and used when more than one kind of object could be present in an application data structure as described in {{Section 4.1.9 of RFC7515}}. If present, the `typ` parameter MUST contain the value `voucher-jws+json`.
+* If X.509 (PKIX) certificates {{RFC5280}} are used, the `x5c` parameter MUST contain the base64-encoded (not base64url-encoded) X.509 v3 (DER) certificate as defined in {{Section 4.1.6 of RFC7515}} and SHOULD also contain the certificate chain.
 
 Implementation Note:
-base64-encoded values opposed to base64url-encoded values may contain slashes ('/').
-JSON {{RFC8259}} optionally allows escaping these with backslashes ('\\').
-Hence, depending on the JSON parser/serializer implementation used, they may or may not be included.
-JWS Voucher parsers must be prepared accordingly to extract certificates correctly.
+: base64-encoded values, in contrast to base64url-encoded values, may contain slashes (`/`).
+  JSON {{RFC8259}} optionally allows escaping these with backslashes (`\\`).
+  Hence, depending on the JSON parser/serializer implementation used, they may or may not be included.
+  JWS Voucher parsers need to be prepared accordingly to extract certificates correctly.
```

Comment:
Implementation note should not have normative statement, hence I replaced the daring "must" with "need to".